add MOLT migration tutorials; simplify Migration Overview (#19408)
cockroachdb/docs: src/current/v25.1/migrate-in-phases.md

	--mode data-load
	~~~
	</section>

	<section class="filter-content" markdown="1" data-scope="mysql">
	{% include_cached copy-clipboard.html %}
	~~~ shell
	molt fetch \
	--source 'mysql://user:password@localhost/molt?sslcert=.%2fsource_certs%2fclient.root.crt&sslkey=.%2fsource_certs%2fclient.root.key&sslmode=verify-full&sslrootcert=.%2fsource_certs%2fca.crt' \
	--target 'postgres://root@localhost:26257/defaultdb?sslmode=verify-full' \
	--table-filter 'employees' \
	--bucket-path 's3://molt-test' \
	--table-handling truncate-if-exists \
	--non-interactive \
	--mode data-load
	~~~
	</section>

{% include molt/fetch-data-load-output.md %}

## Step 4. Verify the data load

{% include molt/verify-output.md %}

Repeat [Step 3](#step-3-load-data-into-cockroachdb) and [Step 4](#step-4-verify-the-data-load) to migrate any remaining tables.

## Step 5. Modify the CockroachDB schema

{% include molt/migration-modify-target-schema.md %}

## Step 6. Replicate changes to CockroachDB

With initial load complete, start replication of ongoing changes on the source to CockroachDB.

The following example specifies that the `employees` table should be watched for change events.

> 1. Issue the [MOLT Fetch]({% link molt/molt-fetch.md %}) command to start replication on CockroachDB, specifying `--mode replication-only` to perform a one-time data load. For details on this mode, refer to the [MOLT Fetch]({% link molt/molt-fetch.md %}#replicate-changes) page.

Author: Nice catch!!

	<section class="filter-content" markdown="1" data-scope="postgres">
	Be sure to specify the same `--pglogical-replication-slot-name` value that you provided in [Step 3](#step-3-load-data-into-cockroachdb).

	{% include_cached copy-clipboard.html %}
	~~~ shell
	molt fetch \
	--source 'postgres://postgres:postgres@localhost:5432/molt?sslmode=verify-full' \
	--target 'postgres://root@localhost:26257/defaultdb?sslmode=verify-full' \
	--table-filter 'employees' \
	--non-interactive \
	--mode replication-only \
	--pglogical-replication-slot-name cdc_slot
	~~~
	</section>

	<section class="filter-content" markdown="1" data-scope="mysql">

> 	Use the `--defaultGTIDSet` replication flag to specify the GTID set. To find your GTID record, run `SELECT source_uuid, min(interval_start), max(interval_end) FROM mysql.gtid_executed GROUP BY source_uuid;` on MySQL.

Reviewer: ```suggestion
	Use the `--defaultGTIDSet` replication flag to specify the GTID set. To find your GTID record, run `SELECT CONCAT(source_uuid,':',min(interval_start),'-',max(interval_end)) FROM mysql.gtid_executed GROUP BY source_uuid;` on MySQL.
```

	{% include_cached copy-clipboard.html %}
	~~~ shell
	molt fetch \
	--source 'mysql://user:password@localhost/molt?sslcert=.%2fsource_certs%2fclient.root.crt&sslkey=.%2fsource_certs%2fclient.root.key&sslmode=verify-full&sslrootcert=.%2fsource_certs%2fca.crt' \
	--target 'postgres://root@localhost:26257/defaultdb?sslmode=verify-full' \
	--table-filter 'employees' \
	--non-interactive \
	--mode replication-only \
	--replicator-flags '--defaultGTIDSet 4c658ae6-e8ad-11ef-8449-0242ac140006:1-29'
	~~~
	</section>

{% include molt/fetch-replication-output.md %}

## Step 7. Stop replication and verify data

{% include molt/migration-stop-replication.md %}

1. Repeat [Step 4](#step-4-verify-the-data-load) to verify the updated data.

{{site.data.alerts.callout_success}}
If you encountered issues with replication, you can now use [`failback`]({% link {{ page.version.version }}/migrate-failback.md %}) mode to replicate changes on CockroachDB back to the initial source database. In case you need to roll back the migration, this ensures that data is consistent on the initial source database.
{{site.data.alerts.end}}

## Step 8. Cutover

> Perform a cutover by resuming application traffic, now to CockroachDB.

Reviewer: ```suggestion
Complete the cutover by resuming application traffic redirected to CockroachDB.
```

Author: I'm going to pass on this suggestion as I believe this step means to initiate & complete the cutover, not just complete a process that had already been ongoing.
